runelite-client/src/main/java/net/runelite/client/ui/overlay/tooltip/TooltipOverlay.java

```diff
@@ -122,6 +129,11 @@ private Dimension renderTooltips(Graphics2D graphics, List<Tooltip> tooltips)
 				mousePosition.move(mouseCanvasPosition.getX(), mouseCanvasPosition.getY() + offset + newBounds.height);
 			}
 
+			if (tooltips.size() > 1 && this.runeLiteConfig.tooltipPosition() == TooltipPositionType.ABOVE_CURSOR)
+			{
+				mousePosition.translate(0, (int) -bounds.getHeight() - offset);
```

Comment:
So this is moving the tooltips back down if they would extend over the top? We don't do this for the normal tooltip layout do we?

Reply:
That is exactly what happens when the tooltip extends under the bottom of the screen, so I replicated it for the top. Although if the tooltip were to extend beyond the right side of the screen it would just stop at the edge.

Reply:
So the original code had different functionalities when going out of the screen if it was either horizontally or vertically